src/mem3/src/mem3_rep.erl

```diff
@@ -152,7 +152,7 @@ filter_hash(_) ->
     <<>>.
 
 local_id_hash(Thing) ->
-    couch_util:encodeBase64Url(couch_hash:md5_hash(term_to_binary(Thing))).
+    couch_util:encodeBase64Url(couch_hash:digest(term_to_binary(Thing))).
```

Comment:
When we switch algorithm, if we're not careful, this will rewind the internal replicator checkpoints.